api/controllers/artists.py

```diff
@@ -33,6 +40,7 @@ def get_artist(name: str, db: Session = Depends(get_db)):
 def create_artist(
     artist: ArtistBase,
     payment_urls: Optional[list[PaymentCreate]],
+    social_links: Optional[list[SocialCreate]],
```

Comment:
How is the media stuff currently working?  Should it be included with the `create_artist` call, or is it strictly a separate thing?